### PR TITLE
Updated for Full Run2

### PR DESCRIPTION
**Pull request by tanmayvb**

### CODE REVIEW

File: python/BeamSpotRcdPrinter_cfg.py [hunk 1]
```diff
@@ -12,13 +12,13 @@
                  "output tag name")
 options.register('startIOV',
                  #1406713458589700, # default value
-                 1173204676640769,
+                 1275820035276812,
```
Comment (mmusich) [inline python/BeamSpotRcdPrinter_cfg.py]: please put a comment this is for 2017 and put back the 2016 one with a comment.

File: python/BeamSpotRcdPrinter_cfg.py [hunk 2]
```diff
@@ -12,13 +12,13 @@
...
                  VarParsing.VarParsing.multiplicity.singleton, # singleton or list
                  VarParsing.VarParsing.varType.int, # string, int, or float
                  "location of the input data")
 options.register('endIOV',
                  #1406876667347162, # default value
-                 1219959690625054,
+                 1316235677532161,
```
Comment (mmusich) [inline python/BeamSpotRcdPrinter_cfg.py]: please put a comment this is for 2017 and put back the 2016 one with a comment.

File: python/BeamSpotScript.py [hunk 3]
```diff
@@ -274,24 +279,34 @@ def make_pdfs(tag='',era='', useMCtruth=False):
...
-    tmc = mc.Get('demo/Events')
-    hmc = ROOT.TH1F('mc_vtxZ','', zbins, zmin, zmax)
-    tmc.Draw('GenVertex_z>>mc_vtxZ') 
     fitmc = ROOT.TF1('fitmc', '[0]/TMath::Sqrt(2*TMath::Pi())/[1]*TMath::Exp( -0.5*(x-[2])*(x-[2])/[1]/[1] )', zmin, zmax)    
-    fitmc.SetParameter(0, hmc.Integral())
-    fitmc.SetParameter(1, hmc.GetRMS())
-    fitmc.SetParameter(2, hmc.GetMean())
-    hmc.Fit(fitmc, 'R')
-    fitmc.SetParameter(0, 1.0)
-    hmc.Scale(1./hmc.Integral())
```
Comment (mmusich) [inline python/BeamSpotScript.py:288]: I would keep this commented.

Comment (tanmayvb) [reply python/BeamSpotScript.py:288]: Yes, but for this we need the test root file.

Comment (mmusich) [reply python/BeamSpotScript.py:288]: not if you keep it commented.

**Comment by tanmayvb** [comment]
@mmusich could you please check and merge it now

**Pull request merged**